app/views/lead_providers/your_schools/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= content_for :before_content, govuk_breadcrumbs(breadcrumbs: {
+  "Dashboard" => dashboard_path,
+  "Your Schools" => nil
+}) %>
+
+<%= render ::SchoolInformation::YourSchools::HeadingComponent.new %>
+<%= render ::SchoolInformation::YourSchools::NavigationComponent.new(cohorts: @cohorts, selected_cohort: @selected_cohort) %>
+<%= render ::SchoolInformation::YourSchools::ParticipantsComponent.new %>
+
+<div class="govuk-grid-row govuk-!-margin-bottom-3">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">See which schools have added their early career teachers and mentors.</p>
```

Comment:
Likewise, `govuk-body` probably not needed.

Probably worth getting rid of it from all paragraphs that have it in pages that inherit from application (unless you notice a difference).

Reply:
There is a difference behind it - it does makes things bigger by adding 20px margin around a paragraph

Reply:
Going to leave it for now